subsys/net/lib/wifi_mgmt_ext/wifi_mgmt_ext.c

```diff
@@ -24,6 +36,69 @@ LOG_MODULE_REGISTER(wifi_mgmt_ext, CONFIG_WIFI_MGMT_EXT_LOG_LEVEL);
 		     "CONFIG_WIFI_CREDENTIALS_STATIC_SSID required");
 #endif /* defined(CONFIG_WIFI_CREDENTIALS_STATIC) */
 
+static int __stored_creds_to_params(struct wifi_credentials_personal *creds,
+				struct wifi_connect_req_params *params)
+{
+	char *ssid = NULL;
+	char *psk = NULL;
+	int ret;
+
+	/* SSID */
+	ssid = (char *) k_malloc(creds->header.ssid_len + 1);
```

Comment:
No space between `(char *)`  and `k_malloc`

Reply:
Done